capabilityreceiver/pkg/capabilityhandler/capabilityreceiver_test.go

```diff
@@ -156,3 +192,30 @@ func Test_formatMessage(t *testing.T) {
 	th.Assert(t, returnErr != nil, "Expected an error to be thrown due to an incorrect smart HTTP response")
 	tmpMessage["smarthttpResponse"] = 200
 }
+
+func Test_RunIncludedFieldsChecks(t *testing.T) {
+	setupCapabilityStatement(t)
+	capInt := testQueueMsg["capabilityStatement"].(map[string]interface{})
+	includedFields := RunIncludedFieldsChecks(capInt)
+	th.Assert(t, includedFields["url"] == true, "Expected url in includedFields to be true, was false")
+	th.Assert(t, includedFields["name"] == true, "Expected name in includedFields to be true, was false")
+	th.Assert(t, includedFields["software.name"] == false, "Expected software.name in includedFields to be false, was true")
+	th.Assert(t, includedFields["format"] == true, "Expected format in includedFields to be true, was false")
+	th.Assert(t, includedFields["contact"] == false, "Expected contact.name in includedFields to be false, was true")
+
+	path := filepath.Join("../../testdata", "wellstar_capability_tester.json")
+	csJSON, err := ioutil.ReadFile(path)
+	th.Assert(t, err == nil, err)
+	var capStat map[string]interface{}
+	err = json.Unmarshal(csJSON, &capStat)
+	th.Assert(t, err == nil, err)
```

Comment:
I think it'd be good to modify the setupCapabilityStatement function to take in a path then call that here with the path to the wellstar capability json

Reply:
I changed the setupCapabilityStatement function so it now takes in a path, and called it with the path to the wellstar capability json now